### PR TITLE
Randomyt

### LINKED ISSUE #1005: Script to return random YouTube videos
# Description

Create a Script to return random YouTube videos. You may use YouTube Data v3 API to achieve the same.

## Language

- [X] Go
- [X] Javascript
- [X] Python

### PR DESCRIPTION
**Pull request by realhunter7869**
# Description

Added script to play a random YouTube video.

Fixes #1005 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by realhunter7869** [comment]
@HarshCasper please merge this Pull Request

File: Python/Random_Youtube_Videos/randomyt.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import string
+import random
+import webbrowser
+import requests
+
+try:
+    API_KEY = str(input("Enter your generated API-KEY : "))
```
Comment (HarshCasper) [inline Python/Random_Youtube_Videos/randomyt.py:7]: Pass the API key from a `.env` file

File: Python/Random_Youtube_Videos/requirement.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+requests
```
Comment (HarshCasper) [inline Python/Random_Youtube_Videos/requirement.txt:1]: Add the version

File: Python/Random_Youtube_Videos/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#### Select the created project in Select a Project
+![Step4](Step4.PNG)
+#### Head to Library section
+![Step5](Step5.PNG)
+#### Search for YouTube Data API v3 and enable it
+![Step6](Step6.PNG)
+#### Click on Create Credentials
+![Step7](Step7.PNG)
+#### Click on API KEY and note your APIKEY (You will require it for searching the video)
+![Step8](Step8.PNG)
```
Comment (HarshCasper) [inline Python/Random_Youtube_Videos/README.md:19]: Remove pictures. Instead use Imgur or any other image hosting site for putting images on the markdown.

File: Python/Random_Youtube_Videos/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Random Yotutbe Videos
+This Python Script fetches random youtube videos on every iteration
```
Comment (HarshCasper) [inline Python/Random_Youtube_Videos/README.md:2]: Follow the standard markdown template provided by us for documentation.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/Random_Youtube_Videos/README.md (hunk 4 above)
Comment (realhunter7869) [reply Python/Random_Youtube_Videos/README.md:2]: Hey, can I know where can I get the standard template?

File: Python/Random_Youtube_Videos/randomyt.py (hunk 1 above)
Comment (realhunter7869) [reply Python/Random_Youtube_Videos/randomyt.py]: Done through .env file... Also provided a sample .env_sample file

File: Python/Random_Youtube_Videos/requirement.txt (hunk 2 above)
Comment (realhunter7869) [reply Python/Random_Youtube_Videos/requirement.txt]: Added versions

File: Python/Random_Youtube_Videos/README.md (hunk 3 above)
Comment (realhunter7869) [reply Python/Random_Youtube_Videos/README.md]: Added imgur pics

File: Python/Random_Youtube_Videos/README.md (hunk 4 above)
Comment (realhunter7869) [reply Python/Random_Youtube_Videos/README.md]: Followed the provided documentation

**Comment by realhunter7869** [comment]
Hey @HarshCasper I have done the requested changes. Please merge the PR.

**Comment by realhunter7869** [comment]
Hey @HarshCasper can you check this PR and merge please... Its already 9 days today... If there are any more changes to be made do let me know....

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**